web/themes/custom/nicsdru_dept_theme/templates/content/node--search-metadata.html.twig

```diff
@@ -3,23 +3,18 @@
  * @file
  * Theme override to display a node in search result view mode.
 #}
-{% set department_name = content.field_domain_source.0["#plain_text"] %}
-{% set content_type = content.field_domain_source["#bundle"] %}
+
 <a class="card card--plain no-ext-icon" href="{{ url }}">
   <h3 class="card__title">{{ label }}</h3>
   <div class="card__summary">
-    {%- if content_type == 'news' %}
-      {{ content.field_teaser }}
-    {% else %}
-      {{ content.field_summary | field_value }}
-    {% endif -%}
+    {{ snippet }}
   </div>
   <div class="card__meta">
-    {%- if department.id == 'nigov' and department_name %}
-      {%- if 'Agriculture' in department_name %}
+    {%- if department.id == 'nigov' and content_domain_source.name %}
+      {%- if content_domain_source.id == 'daera' %}
```

Comment:
I'm wondering if we should have a utility function to turn dept name into friendly name?

Reply:
I'd leave it for now as it's one of the few places the content's underlying source domain is printed but if we have two or more then it would make sense to provide a service to extract and append this to the variables collection so it's there for use in any later templating, a bit like what was done for the current department object